cmu_graphics/cmu_graphics.py

```diff
@@ -249,7 +253,7 @@ def drawFn(*args, **kwargs):
         if (not app._app.inRedrawAll):
             raise MvcException('Cannot draw (modify the view) outside of redrawAll')
         with NoMvc():
-            shape(*args, **kwargs)
+            shape(*args, True, **kwargs)
```

Comment:
I think it would be a little cleaner to set `kwargs['isMvc'] = True` here and then delete it from the kwargs before passing them into slInitShape above on line 97. Or maybe even leaving it in kwargs through slInitShape?

Reply:
I got this method to work, but it also means that someone can call 
`Arc(200, 200, 50, 150, 0, 180, isMvc = True)` and the arc will be drawn with the CPCS guidelines.

I couldn't really find a way around this, unless I'm missing something?

Reply:
I like the new one better (see f9c8704), despite the fact that users can access it. What do you think @austin-schick?